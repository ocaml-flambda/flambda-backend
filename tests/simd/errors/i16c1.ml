open Stdlib

type t = int16x8

external const1 : (int[@untagged]) -> (t[@unboxed]) = "" "caml_int16x8_const1"
        [@@noalloc] [@@builtin]

let _ = const1 (Sys.opaque_identity 1)
