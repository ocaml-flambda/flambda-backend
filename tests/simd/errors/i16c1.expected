File "i16c1.ml", line 1:
Error: Did not find constant int arguments for caml_int16x8_const1
